src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4encoder/HaivisionX4EncoderCommunicator.java

```diff
@@ -269,8 +309,16 @@ public void controlProperty(ControllableProperty controllableProperty) throws Ex
 			logger.debug("controlProperty value" + value);
```

Comment:
do you need this debug entries? 
Also please make sure debug entries are only logged when debug logging is enabled